package.json

```diff
@@ -12,7 +12,7 @@
     "check:package-types": "turbo check:package-types",
     "check:package-lint": "turbo check:package-lint",
     "check:types": "turbo check:types",
-    "publish-packages": "yarn build lint && changeset version && changeset publish && git push --follow-tags origin main",
+    "publish-packages": "yarn build lint && changeset version && yarn turbo build --filter=frames.js... && changeset publish && git push --follow-tags origin main",
```

Comment:
what does `yarn build lint` do? Is it necessary since we have Github Actions in place?

I think it would be enough just to:

1. bump versions
2. build
3. publish
4. push

Reply:
fair enough